packages/fluentui/docs/src/examples/components/Tree/Usage/TreeMultiselectExample.shorthand.steps.ts

```diff
@@ -16,9 +16,9 @@ const config: ScreenerTestsConfig = {
         .snapshot('first expanded, not custom checkbox visible')
         .click(selectors.treeTitle(2))
         .click(selectors.treeTitle(6))
+        .snapshot('default selected states')
         .click(selectors.treeTitle(10))
         .click(selectors.treeTitle(11))
-        .snapshot('default selected states')
```

Comment:
I don't think that this change is correct

Reply:
It's.

Reply:
Solved

Reply:
Reverted